easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.5.0-foss-2016a-NASM-2.12.02.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libjpeg-turbo'
+version = '1.5.0'
+
+homepage = 'http://sourceforge.net/projects/libjpeg-turbo/'
+description = """libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG
+compression and decompression. libjpeg is a library that implements JPEG image encoding, decoding and transcoding.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+nasmver = '2.12.02'
+versionsuffix = '-NASM-%s' % nasmver
```

Comment:
@RvDijk there's no need for this, the existing `libjpeg-turbo-1.5.0-foss-2016a.eb` already depends on `NASM` v2.12.02?